pkg/contexts/ocm/transfer/transfer.go

```diff
@@ -195,10 +236,11 @@ func printArtifactInfo(printer common.Printer, log logging.Logger, kind string,
 		} else {
 			printer.Printf("...%s %d...\n", kind, index)
 		}
-	}
-	if hint != "" {
-		log.Debug(fmt.Sprintf("handle %s", kind), kind, fmt.Sprintf("%d(%s)", index, hint))
 	} else {
-		log.Debug(fmt.Sprintf("handle %s", kind), kind, fmt.Sprintf("%d", index))
+		if hint != "" {
```

Comment:
I think this was double logging. Was that intentional? If printer was defined it would still log again on log.Debug.

Reply:
Hmm, logging is logging and printing is printing. The log messages, if enabled, are sent to a log output. The printer is feedback for the user  (stdout for CLI, or explicit printer setting for a lib user).

Reply:
Okay. I'll leave that as is then. :)